docs/source/references/running.rst

```diff
@@ -0,0 +1,124 @@
+.. _running-the-app:
+
+Successfully Running the DRG
+============================
+
+.. _step-by-step-instr:
+
+Steps to Use the DRG
```

Comment:
I am a little conflicted about this subsection (after reading more of your documentation!). It seems like the steps should be:

```
#. Run the Data Resource Generator -- see :ref:`Run the DRG<running-the-drg-or-whatever-its-called`
#. Create a Data Resource Schema
#. Send your schema to the DRG and generate Data Resources
#. Interact with your generated  data resources
```

I also think each of these steps should have its own page (except steps 2 and 3). (I'll note how to do this on your other RST files.)